### DISCUSSION TITLE
PractitionerRole

### DISCUSSION THREAD

**Discussion by dt-r** [hl7au/au-fhir-core#8] [AU Core R1 Resource Priority List - CLOSED]
Upvote for inclusion of a Resource Profile of [PractitionerRole](http://hl7.org/fhir/R4/practitionerrole.html) Resource in AU Core Release 1.

Comment if you think we need a different profile to AU Core PractitionerRole, or if there is a need for an additional PractitionerRole profile in AU Core.

AU Core PractitionerRole profile would set minimum expectations for a PractitionerRole resource to record, search, and fetch basic demographics and other administrative information about an individual practitioner in a role associated with a patient. 

AU Core PractitionerRole should be based on the [AU Base PractitionerRole ](http://build.fhir.org/ig/hl7au/au-fhir-base/StructureDefinition-au-practitionerrole.html) profile and identify the additional mandatory core elements, extensions, vocabularies and value sets that SHALL be present in the PractitionerRole resource when conforming to this profile. It provides the floor for standards development for specific uses cases in an Australian context.

**Comment by rachelhayhurst** [comment]
Why consider this separately from practitioner - wouldn't they be better together?

**Reply by mmalc** [reply]
Hi @rachelhayhurst , they logically do sit together, and at the face to face session we held in Sydney back in September, we did cover both in the same session.  They are however, separate resources, so we do need to discuss them separately in order to define exactly what needs to be profiled for each of the individual resources.
